content/node-placement/docs/fargate-node-placement.md

```diff
@@ -0,0 +1,178 @@
+# **EKS Fargate Node Placement**
+## **Fargate Node Placement**
+
+AWS Fargate is a technology that provides on-demand, right-sized compute capacity for containers. With AWS Fargate, you don't have to provision, configure, or scale groups of EC2 instances on your own to run containers. You also don't need to choose server types, decide when to scale your node groups, or optimize cluster packing. Instead you can control which pods start on Fargate and how they run with Fargate profiles.
+
+## AWS Fargate profile 
+Before you can schedule pods on Fargate in your cluster, you must define at least one Fargate profile that specifies which pods use Fargate when launched.  You must define a namespace for every selector. The Fargate profile allows an administrator to declare which pods run on Fargate. This declaration is done through the profile’s selectors. If a namespace selector is defined without any labels, Amazon EKS attempts to schedule all pods that run in that namespace onto Fargate using the profile.
+
+A Spark application whose driver and executor pods are distributed across multiple AZs can incur inter-AZ data transfer costs. To minimize or eliminate inter-AZ data transfer costs, you can configure the application to only run on the nodes within a single AZ.  In this example, we use the kubernetes [node selector](https://spark.apache.org/docs/latest/running-on-kubernetes.html#pod-spec) to specify which AZ should the job run on.
+
+
+**Create Fargate Profile**
+Create your Fargate profile with the following eksctl command, replacing the `<variable text>` (including <>) with your own values. You're required to specify a namespace. However, the `--labels` option is not required.
+
+```
+eksctl create fargateprofile \
+    --cluster <cluster_name> \
+    --name <fargate_profile_name> \
+    --namespace <virtual_cluster_mapped_namespace> \
+    --labels spark-node-placement=fargate
+```
+
+### 1- Place entire job including driver pod on Fargate 
+
+When both Driver and Executor are labeled that matches the Fargate Selector, the entire job including the driver pod could run on Fargate.
+
+**Request:**
+```
+cat >spark-python-in-s3-nodeselector.json << EOF
+{
+  "name": "spark-python-in-s3-fargate-nodeselector", 
+  "virtualClusterId": "<virtual-cluster-id>", 
+  "executionRoleArn": "<execution-role-arn>", 
+  "releaseLabel": "emr-6.3.0-latest", 
+  "jobDriver": {
+    "sparkSubmitJobDriver": {
+      "entryPoint": "s3://<s3 prefix>/trip-count.py", 
+       "sparkSubmitParameters": "--conf spark.driver.cores=4  --conf spark.executor.memory=20G --conf spark.driver.memory=20G --conf spark.executor.cores=4"
```

Comment:
Fargate can only support 3 cores, right?

```suggestion
       "sparkSubmitParameters": "--conf spark.driver.cores=3  --conf spark.executor.memory=20G --conf spark.driver.memory=20G --conf spark.executor.cores=3"
```

Reply:
Actually, Fargate can support up to 4 vCPUs https://docs.aws.amazon.com/AmazonECS/latest/developerguide/AWS_Fargate.html#fargate-tasks-size
I tested the above configurations and Driver and Executors both ran successfully all on Fargate.